modules/openxr/openxr_api.cpp

```diff
@@ -588,7 +585,9 @@ bool OpenXRAPI::create_instance() {
 		extension_ptrs.ptr() // enabledExtensionNames
 	};
 
-	copy_string_to_char_buffer(project_name, instance_create_info.applicationInfo.applicationName, XR_MAX_APPLICATION_NAME_SIZE);
+	if (project_name != "") {
```

Comment:
```suggestion
	if (!project_name.is_empty()) {
```

Also, I'd suggest moving the `String project_name = ...` call before this, as it doesn't seem to be used in between so it's weird for it to be separated this way.

Reply:
Fixed!